connector/connect/src/main/protobuf/spark/connect/relations.proto

```diff
@@ -404,6 +405,18 @@ message StatSummary {
   repeated string statistics = 2;
 }
 
+// Computes basic statistics for numeric and string columns, including count, mean, stddev, min,
+// and max. If no columns are given, this function computes statistics for all numerical or
+// string columns.
+// It will invoke 'Dataset.describe' to compute the results.
+message StatDescribe {
+  // (Required) The input relation.
+  Relation input = 1;
+
+  // Columns to compute statistics on.
+  repeated string cols = 2;
```

Comment:
Can you follow https://github.com/apache/spark/blob/master/connector/connect/docs/adding-proto-messages.md to mark if this is required or optional?

Reply:
I got it.